sdk/js/src/CacheStore.ts

```diff
@@ -24,17 +30,53 @@ export abstract class CacheStore {
         return this.store?.getItem(storeKey)
     }
 
+    private getConfigStoreKey(user: DVCPopulatedUser) {
+        return user.isAnonymous ? StoreKey.AnonymousConfig : StoreKey.IdentifiedConfig
+    }
+
+    private getUserIdStoreKey(user: DVCPopulatedUser) {
```

Comment:
This key is specific to the config, so we should include that in the name. We also probably don't need to include "store" since everything in this class is scoped to the cache store
```suggestion
    private getConfigUserIdKey(user: DVCPopulatedUser) {
```

Reply:
yup, I feel the same, not good at naming :(, thanks for the suggestion!